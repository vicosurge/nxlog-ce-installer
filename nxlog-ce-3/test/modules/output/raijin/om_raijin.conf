include tmp/common.conf

<Extension jsonmodule>
  Module  xm_json
</Extension>

# raw_event is absent [test-case]

<Input http>
    Module              im_tcp
    Host                127.0.0.1
    Port                8888
</Input>

<Output outh>
    Module om_file
    File   "tmp/output"
</Output>

<Route r2>
    Path    http => outh
</Route>


<Input in1>
  Module im_file
  File  "modules/output/raijin/om_raijin.in"
  ReadFromLast False
  SavePos  False

  <Exec>
    delete($EventReceivedTime);
    delete($SourceModuleName);
    delete($SourceModuleType);

    delete($raw_event);
    $new_event = "raw_event has been deleted";
    to_json();
  </Exec>
</Input>

<Output out1>
    Module              om_raijin
    URL                 http://127.0.0.1:8888/
    DBName nxlogDB
    DBTable logsTable
</Output>

<Route r1>
    Path    in1 => out1
</Route>


