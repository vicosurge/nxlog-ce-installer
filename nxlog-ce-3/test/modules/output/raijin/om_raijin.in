{ "id": 45, "first_name": "Alex", "last_name": "Vort", "active": true }
