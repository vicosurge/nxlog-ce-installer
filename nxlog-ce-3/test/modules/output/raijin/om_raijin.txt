REMOVE: tmp/om_raijin.in
REMOVE: tmp/output

STARTDAEMON: modules/output/raijin/om_raijin.conf
SLEEP: 9
STOPDAEMON: modules/output/raijin/om_raijin.conf
COMPAREFILE: tmp/output modules/output/raijin/om_raijin.out

REMOVE: tmp/om_raijin.in
REMOVE: tmp/output
