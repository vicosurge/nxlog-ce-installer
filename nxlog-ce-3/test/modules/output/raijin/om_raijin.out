POST /nxlogDB/logsTable HTTP/1.1
User-Agent: nxlog-ce
Host: 127.0.0.1:8888
Content-Type: application/json; charset=utf-8
Connection: Keep-Alive
Keep-Alive: 300
Content-Length: 43

{"new_event":"raw_event has been deleted"}
